packages/common-utils/src/types.ts

```diff
@@ -346,6 +347,7 @@ export const SavedChartConfigSchema = z.intersection(
     z.object({
       name: z.string(),
       source: z.string(),
+      alert: AlertSchemaBase.optional(),
```

Comment:
isn't this `AlertSchema` ?

Reply:
not really, since tile alerts are managed via `/dashboard` routes, we already know its source (Tile) and `dashboardId` and `tileId`, so we only need alert configuration here

Reply:
Got it. But don't we still attach the redundant fields? from `getDashboardAlerts`, we fetch all fields in the alert. I guess the same thing applied to https://github.com/hyperdxio/hyperdx/blob/f1228ded84069b2bc73987f1e3e4b49cf9bbc5d6/packages/common-utils/src/types.ts#L269